setup.py

```diff
@@ -3,7 +3,7 @@
 
 setup(
     name='hm_pyhelper',
-    version='0.8.4',
+    version='0.8.6',
```

Comment:
I will bump this after a final review. Otherwise it will result in several unnecessary bumps, see: https://github.com/NebraLtd/hm-pyhelper/issues/20